libraries/aws_ec2_traffic_mirror_target.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require 'aws_backend'
+
+class AWSEc2TrafficMirrorTarget < AwsResourceBase
+  name 'aws_ec2_traffic_mirror_target'
+  desc 'Audits EC2 Traffic Mirror target'
```

Comment:
Change the description to "Information about one or more Traffic Mirror targets."